binary_port/src/type_wrappers.rs

```diff
@@ -173,6 +173,55 @@ impl GetTrieFullResult {
     }
 }
 
+/// Type representing the reward of a validator or a delegator.
+#[derive(Debug, PartialEq, Eq)]
+pub struct RewardResponse {
+    amount: U512,
```

Comment:
right now it's unambigous whose reward it is by looking at the request (if delegator is `Some` it's theirs, otherwise validators), so I'll leave it like this

Reply:
we could extend the response on the sidecar-side if we need to since it can even be inferred there

Reply:
if we add a fancier kind of request in the future, it'll probably deserve it's own response type anyway